packages/authentication/services/auth.ts

```diff
@@ -10,6 +10,10 @@ import {
 
 export default class AuthApi {
   static login({ email, password }: ILoginRequest): Promise<LoginResponse> {
```

Comment:
I agree, I'll see if I can update the templates quickly and make all the endpoints in the auth package be like that

Reply:
Did it only for the `/login` one, since the other ones are for account management and should be under `/account` or something like that, but that's outside the scope of this story.